app/src/js/components/AddRaw/add-raw.js

```diff
@@ -37,6 +37,16 @@ const AddRaw = ({
 
   const handleOnClick = requireConfirmation ? handleModalOpen : handleSubmit;
 
+  useEffect(() => {
+    setRecord(record => {
+      const data = JSON.stringify(defaultValue, null, 2);
+      return {
+        ...record,
+        data
+      };
+    });
+  }, [defaultValue]);
+
```

Comment:
I know that the call to `useEffect` on line 50 is not part of your commits, but when I run `npm run lint`, I get the following warning. Perhaps you can implement the suggested fix to eliminate the warning.
```
/Users/chuck/src/nasa/cumulus-dashboard/app/src/js/components/AddRaw/add-raw.js
  50:3  warning  React Hook useEffect contains a call to 'setRecord'. Without a list of dependencies, this can lead to an infinite chain of updates. To fix this, pass [pk, state.created, error, getBaseRoute, history, record] as a second argument to the useEffect Hook  react-hooks/exhaustive-deps
```

Reply:
Done

Reply:
What happened to the list of deps?